dev/com.ibm.websphere.org.eclipse.microprofile.config.1.4/src/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -28,6 +28,8 @@
  *******************************************************************************/
 package org.eclipse.microprofile.config.spi;
 
+import java.util.Collections;
+import java.util.HashMap;
```

Comment:
Are these imports needed?

Reply:
No, but it doesn't matter, this is temporary API source. Before GA this will be replaced by the binary API jar from maven central.